### DISCUSSION TITLE
Is it possible to integrate more rooms?

### DISCUSSION THREAD

**Discussion by Trurl101** [dolezsa/thermal_comfort#138] [Ideas]
It just installed it from Hacs and it seems to work flawlessly.

If have only one question: Is it possible to integrate more rooms or can I use the integration only for one room/set of sensors?

**Comment by rautesamtr** [comment]
I assume you are asking for config flow configuration? You just have to add an additional integration per room by clicking again at `add integration` and configure a new set of input sensors.

**Reply by Trurl101** [reply]
> I assume you are asking for config flow configuration? You just have to add an additional integration per room by clicking again at `add integration` and configure a new set of input sensors.

Thanks for your answer. Now it works. I did the exact same thing before and "Thermal Integration" did not show up in the Integration list after I added the first room. Now it shows up again and I can add more rooms.
